spec/capybara_helper.rb

```diff
@@ -2,16 +2,18 @@
 require 'capybara/rails'
 require 'capybara/poltergeist'
 require 'capybara-select2'
+require 'helpers/capybara_poltergeist_screenshot'
 
 Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, timeout: 60)
+  Capybara::Poltergeist::Driver.new(app, timeout: 5)
```

Comment:
I reran the Travis job two or three times and got the first timeout error, maybe we can make it dependent on `ENV['CI']`? The long timeout is annoying me as well when writing tests :)